internal/venice-test-common/src/main/java/com/linkedin/venice/utils/InMemoryLogAppender.java

```diff
@@ -0,0 +1,41 @@
+package com.linkedin.venice.utils;
```

Comment:
Could we move this class into a test package?  This thing makes me a little nervous, wouldn't mind shuffling it away somewhere where it's unlikely to hurt anybody.

Reply:
Nevermind, I read the package name wrong.